packages/button/src/Button.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { mount } from "enzyme";
+import Button from "./Button";
+
+const subject = (props = {}) => mount(<Button title="Button" {...props} />);
```

Comment:
Lets move this inside the `describe("Button"` and use a `beforeEach` to create a new instance for each test.